package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "express-generator",
   "description": "Express' application generator",
-  "version": "4.15.0",
+  "version": "4.16.0",
```

Comment:
There is no Express.js 4.16.0; this version matches the version of Express. Just remove the version bump from the PR.